File: Compose-My-Pohang-App/app/src/main/java/com/example/mypohangapp/model/Recommend.kt
package com.example.mypohangapp.model

import androidx.annotation.DrawableRes
import androidx.annotation.StringRes

data class Recommend(
    val categoryType: CategoryType,
    @StringRes val name: Int,
    @StringRes val location: Int,
    @StringRes val information: Int,
    @StringRes val source: Int,
    @DrawableRes val picture: Int
)

File: SwiftUI-My-Pohang-App/My Pohang App/Model/Recommend.swift
//
//  Recommend.swift
//  My Pohang App
//
//  Created by Jaehwa Noh on 12/22/23.
//

struct Recommend: Identifiable {
    let categoryType: CategoryType
    let name: String
    let location: String
    let information: String
    let source: String = "한국관광공사"
    let picture: String
    
    var id: String {
        categoryType.rawValue + name
    }
}
